### DISCUSSION TITLE
Ability to set keybinds to switch Tournament scenes and / or button actions on spesific scenes.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#17475] [Ideas] [answered]
As above, when doing streams using the tournament client, you would normally have the scene changes automatic. However, there are times where the scenes can't change (i.e. from Gameplay to Mappool) and you don't want to move your mouse to the spesific tab.
So an idea I'm thinking of is to allow the streamer to add keybinds for changing scenes (i.e. `m` for Mappool and `p` / `g` for gameplay... etc.) This could be used with a KeyBindingsPanel that is edited to only show Tournament and ruleset related keybinds like below:
![image](https://user-images.githubusercontent.com/37494321/160267384-fa28a4c5-515e-4e3e-a390-a41edc376bdd.png)

I think this would make streaming a lot more streamline without the use of the mouse for the majority of a match.

**Comment by peppy** [comment, marked as answer]
It would be easy to implement these if they don't require customisation. Would you be okay with that (using the first letter of each of the screen names)?

**Reply by Coppertine** [reply]
would be possible for now.

**Reply by peppy** [reply]
Added in #17589.
